app/managers.py

```diff
@@ -0,0 +1,59 @@
+import sqlite3
+
+from app.models import Actor
+
+
+class ActorManager:
+    def __init__(self) -> None:
+        self.table_name = "actors"
+        self._connection = sqlite3.connect("cinema.sqlite")

```

Comment:
Consider closing the database connection when it's no longer needed to prevent potential database locks or resource leaks. You might want to implement a method to close the connection or use a context manager.